Platform metadata credentials (#8950)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

## April 2026

> - **Enhancement:** When a connection does not have platform metadata credentials configured yet, the credentials form now renders in edit mode immediately &mdash; you no longer need to click **Add credentials** first. If you cancel, the **Add credentials** button appears so you can return to the form. Existing connections with configured platform metadata credentials are unaffected.

Reviewer: Suggest we add a link to the related doc (External metadata ingestion) for this release note.

> - **Enhancement:** When a connection does not have platform metadata credentials configured yet, the credentials form now renders in edit mode immediately &mdash; you no longer need to click **Add credentials** first. If you cancel, the **Add credentials** button appears so you can return to the form. Existing connections with configured platform metadata credentials are unaffected.

Author: ```suggestion
- **Enhancement:** When a connection does not have platform metadata credentials configured yet, the credentials form now renders in edit mode immediately &mdash; you no longer need to click **Add credentials** first. If you cancel, the **Add credentials** button appears so you can return to the form. Existing connections with configured platform metadata credentials are unaffected. Refer to the [Configure the warehouse connection](/docs/explore/external-metadata-ingestion) section for more information.
```

> - **Enhancement:** When a connection does not have platform metadata credentials configured yet, the credentials form now renders in edit mode immediately &mdash; you no longer need to click **Add credentials** first. If you cancel, the **Add credentials** button appears so you can return to the form. Existing connections with configured platform metadata credentials are unaffected. Refer to [External metadata ingestion](/docs/explore/external-metadata-ingestion) for more information.

Author: ```suggestion
- **Enhancement:** When a connection does not have platform metadata credentials configured yet, the credentials form now renders in edit mode immediately &mdash; you no longer need to click **Add credentials** first. If you cancel, the **Add credentials** button appears so you can return to the form. Existing connections with configured platform metadata credentials are unaffected. Refer to [Configure the warehouse connection](/docs/explore/external-metadata-ingestion#configure-the-warehouse-connection) for more information.
```

> - **Enhancement:** When a connection does not have platform metadata credentials configured yet, the credentials form now renders in edit mode immediately &mdash; you no longer need to click **Add credentials** first. If you cancel, the **Add credentials** button appears so you can return to the form. Existing connections with configured platform metadata credentials are unaffected. Refer to the [Configure the warehouse connection](/docs/explore/external-metadata-ingestion) section for more information.

Author: ```suggestion
- **Enhancement:** When a connection does not have platform metadata credentials configured yet, the credentials form now renders in edit mode immediately &mdash; you no longer need to click **Add credentials** first. If you cancel, the **Add credentials** button appears so you can return to the form. Existing connections with configured platform metadata credentials are unaffected. Refer to [External metadata ingestion](/docs/explore/external-metadata-ingestion) for more information.
```

- **Enhancement:** The Studio IDE status bar now offers more control, more detailed information, and quicker access to settings for deferral, dbt version, and project status. For more information, refer to the [Studio IDE docs](/docs/cloud/studio-ide/ide-user-interface#the-command-and-status-bar). These updates roll out in phases to existing accounts starting April 6.
- **Enhancement:** In Snowflake **Private endpoints**, output validation errors now display inline beneath the text area (instead of as a page-level banner). The **Submit request** button is also disabled when the output is invalid (for example, empty, malformed JSON, or missing required fields).

> - **Enhancement:** The platform metadata credentials setup is now more discoverable. When a connection does not yet have platform metadata credentials configured, the credentials form now opens by default, so you no longer need to click **Add credentials** first. If you cancel setup, the **Add credentials** option still appears so you can reopen the form. Existing connections with configured credentials keep their current edit flow.

Reviewer: Please move this to the top -- we add new release notes at the top of the list

> - **Enhancement:** The platform metadata credentials setup is now more discoverable. When a connection does not yet have platform metadata credentials configured, the credentials form now opens by default, so you no longer need to click **Add credentials** first. If you cancel setup, the **Add credentials** option still appears so you can reopen the form. Existing connections with configured credentials keep their current edit flow.

Reviewer: Saying "opens by default" could mean that there is a modal or form that pops up
```suggestion
- **Enhancement:** When a connection does not have platform metadata credentials configured yet, the credentials form now renders in edit mode immediately &mdash; you no longer need to click **Add credentials** first. If you cancel, the **Add credentials** button appears so you can return to the form. Existing connections with configured platform metadata credentials are unaffected.
```
